### PR TITLE
Folder change

### PR DESCRIPTION
**Pull request by AlbertoSoutullo**
Small PR to improve prepare_path function

Depends on #36

### CODE REVIEW

**Comment by AlbertoSoutullo** [comment]
> > Small PR to improve prepare_path function
> > Depends on #36
> 
> Could you temporarily set this branch to merge into the one corresponding to #36, so I get the "correct" diff? Just so I don't miss any lines by mistake.

My bad, I should have not put you as reviewer if the PR was still in Draft.
I filled it completely afk brain. My intention was to rebase it. Now it is ready.

File: src/mesh_analysis/waku_message_log_analyzer.py [hunk 1]
```diff
@@ -29,7 +29,12 @@ def _validate_analysis_location(self, timestamp_to_analyze: str, local_folder_to
             exit(1)
 
     def _set_up_paths(self, dump_analysis_dir: str, local_folder_to_analyze: str):
-        self._folder_path = Path(dump_analysis_dir) if dump_analysis_dir else Path(local_folder_to_analyze)
+        self._dump_analysis_dir = Path(dump_analysis_dir) if dump_analysis_dir else None
+        self._local_path_to_analyze = Path(local_folder_to_analyze) if local_folder_to_analyze else None
+        result = path_utils.prepare_path_for_folder(self._dump_analysis_dir)
+        if result.is_err():
+            logger.error(result.err_value)
+            exit(1)
```
Comment (AlejandroCabeza) [inline src/mesh_analysis/waku_message_log_analyzer.py:37]: Has this reverted?

File: src/mesh_analysis/waku_message_log_analyzer.py [hunk 2]
```diff
@@ -84,7 +89,7 @@ def _get_affected_node_log(self, data_file: str) -> Result[Path, str]:
         pod_log = reader.read()
 
         log_lines = [inner_list[0] for inner_list in pod_log[0]]
-        log_name_path = self._folder_path / f"{data_file.split('.')[0]}.log"
+        log_name_path = self._dump_analysis_dir / f"{data_file.split('.')[0]}.log"
```
Comment (AlejandroCabeza) [inline src/mesh_analysis/waku_message_log_analyzer.py:92]: Same thing here. Hadn't you changed the two dir attributes for the `self._folder_path`?

**Review by AlejandroCabeza — APPROVED** [review]

File: src/mesh_analysis/waku_message_log_analyzer.py (hunk 2 above)
Comment (AlbertoSoutullo) [reply src/mesh_analysis/waku_message_log_analyzer.py:92]: Yes, and then I realized it was dump. And split it again. One for reading data and another one for dumping results. That makes much more sense...

**Pull request merged**